### PR TITLE
Refactor onboarding and split in subpages

### PR DESCRIPTION
**Pull request by CHrlS98**
Refactor onboarding page. Simpler and shorter bullet points a new student must go through. Move sections on computer setup to new page.

### CODE REVIEW

File: source/arriving/remote_work.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.. _ref_remote_work:
+
+Working from home
```
Comment (karanphil) [inline source/arriving/remote_work.rst:3]: In my opinion, "working from home" should be a section of "Setting up your computer", possibly named "SSH" or something.

File: source/arriving/welcome.rst [hunk 2]
```diff
@@ -5,86 +5,35 @@ Onboarding new students
```
Comment (karanphil) [inline source/arriving/welcome.rst:5]: Should we add a small welcome paragraph here?

File: source/arriving/welcome.rst [hunk 3]
```diff
@@ -5,86 +5,35 @@ Onboarding new students
 
 Important rooms and places
 """"""""""""""""""""""""""
-Chantal Proulx, Marie-Pier Côté and generally the `Faculty’s offices <https://www.usherbrooke.ca/informatique/nous-joindre/personnel>`_ are important and you should know where they are. Ask someone in the lab or Maxime to guide you.
+The Sherbrooke Connectivity Imaging Laboratory is located in the Faculty of Sciences of the University of Sherbrooke (room number **D6-0051**). Other important rooms include the Faculty’s offices:
```
Comment (karanphil) [inline source/arriving/welcome.rst:8]: Maybe add François and Maxime rooms?

**Review by karanphil — CHANGES_REQUESTED** [review]
GTG after some small changes!

File: source/arriving/welcome.rst (hunk 3 above)
Comment (karanphil) [reply source/arriving/welcome.rst:8]: According to the UdeS page, François is at D4-1010-5 and Maxime is at D4-1016-2.

File: source/arriving/welcome.rst (hunk 2 above)
Comment (CHrlS98) [reply source/arriving/welcome.rst:5]: There is a welcome message in `index.rst` which is the first page loaded. We might want to rewrite it but can be a later TODO

File: source/arriving/welcome.rst [hunk 4]
```diff
@@ -3,88 +3,39 @@
...
 
-Mailing list
-    When Maxime wants to send a message to everyone, we either use Teams or our mailing list. Ask Maxime to add you.
+* | **Get your keys to the lab**
+  | Your supervisor will ask them for you. You should receive an email.
 
-Teams
-    We use Microsoft Teams as means of communication. You can use your CIP and university's password to connect. You can ask Maxime to add you to the SCIL's "team". Then, you can explore the various conversations, called "channels". When lab meetings are online, you can join them via the "General" channel. Also, we encourage you to explore the videos in the video tab from the General channel. You can find there the recording of previous lab meetings, videos of Max's dMRI class and more.
+* | **Ask to be added to the SCIL Team**
+  | You can connect to the SCIL Team on Microsoft Teams with your CIP and university's password.
```
Comment (karanphil) [inline source/arriving/welcome.rst:26]: University password

**Review by karanphil — COMMENTED** [review]
Could you change this small detail?

**Review by karanphil — APPROVED** [review]

**Pull request merged**